### PR TITLE
Editor: Add storybook story for icons

### PR DESCRIPTION
**Pull request by swissspidy**
## Summary

This brings the icons library in the editor on par with the dashboards', making it easier to import them and test them in storybook.

## User-facing changes

None

## Testing Instructions

1. `npm run storybook`
1. Check out icons
1 In WordPress, open the editor
1. Verify nothing is broken

### CODE REVIEW

**Comment by miina** [comment]
@swissspidy Two findings so far (or perhaps it's the same finding twice):
1. Console errors when opening the grid view:
Screenshot 2020-06-03 at 11 05 58

2. One of the zoom icons missing:
Screenshot 2020-06-03 at 11 06 02

File: assets/src/edit-story/components/canvas/gridview/index.js [hunk 1]
```diff
@@ -201,7 +201,7 @@ function ThumbnailSizeControl({ value, onChange }) {
         disabled={value === max}
         aria-label={__('Increase thumbnail size', 'web-stories')}
       >
-        <RectangleIcon />
+        <Rectangle />
```
Comment (swissspidy) [inline assets/src/edit-story/components/canvas/gridview/index.js]: To fix the issue @miina found:

```suggestion
        <RectangleIcon />
```

**Comment by swissspidy** [comment]
@miina fixed that particular issue

**Review by miina — APPROVED** [review]
LGTM

**Pull request merged**